app/controllers/api/v1/offers_controller.rb

```diff
@@ -65,14 +65,14 @@ def index
         @options = { each_serializer: select_serializer, include_orders_packages: false,
           exclude_messages: params["exclude_messages"] == "true", root: 'offers' }
         @options.merge!(summarize: true) if params[:summarize] == 'true'
-        render json: @offers, **@options
+        render json: ActiveModel::ArraySerializer.new(@offers.with_summary_eager_load, **@options).as_json
       end
 
       api :GET, '/v1/offers/search?searchText=xyz', "Search for offers"
       def search
         records = @offers.search({ search_text: params['searchText'], states: array_param(:state) })
         records = apply_filters(records)
-        records = records.page(params["page"]).per(params["per_page"] || DEFAULT_SEARCH_COUNT)
+        records = records.page(params["page"]).per(params["per_page"] || params["recent_offer_count"] || DEFAULT_SEARCH_COUNT)
```

Comment:
@steveyken , I am doing this change today and it will take today's time to complete it. 

Let me know should go with release without this or do the release tomorrow ?

cc:  @namrataukirde